### ISSUE TITLE
Unhandled Runtime Error Error: Hydration failed

### ISSUE THREAD

**Issue by newbNox** [Maverick-Security/gamerep-site#2]
I believe something to do with the index routing, there is currently Unhandled Runtime Error:
```
Unhandled Runtime Error
Error: Hydration failed because the initial UI does not match what was rendered on the server.

Warning: Expected server HTML to contain a matching <html> in <div>.

See more info here: https://nextjs.org/docs/messages/react-hydration-error

Component Stack
html
RootLayout
MyApp
```

**Comment by korhox** [comment]
Fixed in PR #3

**Comment by newbNox** [comment]
PR #3 fixed the issue
